edn/src/types.rs

```diff
@@ -248,7 +389,32 @@ impl Ord for Value {
     }
 }
 
+impl PartialOrd for SpannedValue {
+    fn partial_cmp(&self, other: &SpannedValue) -> Option<Ordering> {
+        Some(self.cmp(other))
+    }
+}
+
+impl Ord for SpannedValue {
+    fn cmp(&self, other: &SpannedValue) -> Ordering {
+        Value::from(self.clone()).cmp(&Value::from(other.clone()))
```

Comment:
This is going to be very expensive.  Why do either of these types implement `Ord` and `PartialOrd` at all?  What is the desired ordering?  Is it a domain concept?  Is it because we're using `edn::Value` in some ordered data structure?  Maybe the inner `BTreeSet` and `BTreeMap`?  We might not want to do that.

Can we be clear on what we're doing here and why?